### ISSUE TITLE
Multiple Instance of Bitcore lib found - Nestjs/Serverless Offline

### ISSUE THREAD

**Issue by CITGuru** [tatumio/tatum-js#61]
Error: More than one instance of bitcore-lib found. Please make sure to require bitcore-lib and check that submodules do not also include their own bitcore-lib dependency.

Thats the error i am getting when running this in Serverless/Lambda mode. Currently using Nestjs for serverless functions and I only get this while running it locally using serverless.

```
    at Object.bitcore.versionGuard (/Users/mac/Bitpowr/walletsolution/node_modules/bitcore-lib/index.js:12:11)
    at Object.<anonymous> (/Users/mac/Bitpowr/walletsolution/node_modules/bitcore-lib/index.js:15:9)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/Users/mac/Bitpowr/walletsolution/node_modules/@tatumio/tatum/src/offchain/bitcoin.ts:3:1)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/Users/mac/Bitpowr/walletsolution/node_modules/@tatumio/tatum/src/offchain/index.ts:2:1)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/Users/mac/Bitpowr/walletsolution/node_modules/@tatumio/tatum/src/index.ts:7:1)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/Users/mac/Bitpowr/walletsolution/src/enums/assets.ts:1:1)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/Users/mac/Bitpowr/walletsolution/src/modules/assets/service.ts:3:1)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/Users/mac/Bitpowr/walletsolution/src/modules/wallets/controller.ts:11:1)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/Users/mac/Bitpowr/walletsolution/src/modules/wallets/module.ts:2:1)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/Users/mac/Bitpowr/walletsolution/src/modules/wallets/lambda.ts:5:1)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at /Users/mac/Bitpowr/walletsolution/node_modules/serverless-offline/dist/lambda/handler-runner/in-process-runner/InProcessRunner.js:157:133
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at InProcessRunner.run (/Users/mac/Bitpowr/walletsolution/node_modules/serverless-offline/dist/lambda/handler-runner/in-process-runner/InProcessRunner.js:157:9)```

**Comment by isra67** [comment]
Could you provide your package.json file?

**Comment by CITGuru** [comment]
```"dependencies": {
    "@nestjs/common": "^7.6.15",
    "@nestjs/core": "^7.6.15",
    "@nestjs/platform-express": "^7.6.15",
    "@prisma/client": "^2.22.1",
    "@tatumio/tatum": "^1.13.13",
    "@types/bcryptjs": "^2.4.2",
    "akeyless_vault_api": "akeylesslabs/akeyless-javascript-sdk",
    "aws-lambda": "^1.0.6",
    "aws-sdk": "^2.909.0",
    "aws-serverless-express": "^3.4.0",
    "axios": "^0.21.1",
    "bclient": "^0.1.7",
    "bcryptjs": "^2.4.3",
    "bwallet": "^0.0.0",
    "class-validator": "^0.13.1",
    "crypto-js": "^4.0.0",
    "dotenv": "^9.0.2",
    "reflect-metadata": "^0.1.13",
    "rimraf": "^3.0.2",
    "rxjs": "^6.6.6",
    "serverless-dotenv-plugin": "^3.9.0"
  },
  "devDependencies": {
    "@hewmen/serverless-plugin-typescript": "^1.1.17",
    "@nestjs/cli": "^7.6.0",
    "@nestjs/schematics": "^7.3.0",
    "@nestjs/testing": "^7.6.15",
    "@types/express": "^4.17.11",
    "@types/jest": "^26.0.22",
    "@types/node": "^14.14.36",
    "@types/supertest": "^2.0.10",
    "@typescript-eslint/eslint-plugin": "^4.19.0",
    "@typescript-eslint/parser": "^4.19.0",
    "copy-webpack-plugin": "^8.1.1",
    "eslint": "^7.22.0",
    "eslint-config-prettier": "^8.1.0",
    "eslint-plugin-prettier": "^3.3.1",
    "jest": "^26.6.3",
    "plugin": "^0.3.3",
    "prettier": "^2.2.1",
    "prisma": "^2.22.1",
    "serverless-offline": "^7.0.0",
    "serverless-plugin-optimize": "^4.2.1-rc.1",
    "serverless-plugin-typescript": "^1.1.9",
    "serverless-webpack": "^5.5.0",
    "supertest": "^6.1.3",
    "ts-jest": "^26.5.4",
    "ts-loader": "^8.0.18",
    "ts-node": "^9.1.1",
    "tsconfig-paths": "^3.9.0",
    "typescript": "^4.2.3",
    "webpack-node-externals": "^3.0.0"
  },```

**Comment by isra67** [comment]
Fixed: use version 1.13.15.
Thanks for reporting.
